ferrocene/doc/internal-procedures/src/backports.rst

```diff
@@ -104,3 +104,29 @@ GitHub for review.
 Note that the manual backport script also adds the correct metadata needed for
 :ref:`backport-phase-cleanup` to run: as long as you use the script, you will
 not need to manually remove the backport labels from the PR.
+
+Manual Backports of rust-lang/rust PRs
+--------------------------------------
+
+To backport a PR from ``rust-lang/rust``, a bit of initial one-time preparation is needed. You may
+skip the following step if it you have already done this once in your ferrocene checkout.
+
+.. code-block::
+   # Add a remote to rust-lang/rust
+   git remote add upstream-rust https://github.com/rust-lang/rust.git
+   # Alternatively, add it as an ssh remote
+   # git remote add upstream-rust git@github.com:rust-lang/rust.git
+   git fetch upstream-rust
```

Comment:
We can avoid this by having the backport script fetch the commit it's going to backport before running.

Reply:
Oh, that is a good idea